### PR TITLE
Dynamic Icons based on field data

### PR DESCRIPTION
**Pull request by Dalton098**
Dynamically load icons based off the forecasted weather

### CODE REVIEW

File: frontend/Weather101/src/app/app.component.html [hunk 1]
```diff
@@ -4,22 +4,16 @@
...
         <mat-icon class="right-selection-item">menu</mat-icon>
     </mat-toolbar-row>
-    <mat-toolbar-row class="title-selection fira-code">
-        <button mat-button id="title-today" [routerLink]="['/daily']" class="title-selection-item">Today</button>
-        <button mat-button id="title-hourly" [routerLink]="['/hourly']" class="title-selection-item">Hourly</button>
-        <button mat-button id="title-weekly" class="title-selection-item">Weekly</button>
-        <button mat-button id="title-monthly" class="title-selection-item">Monthly</button>
+    <mat-toolbar-row class="title-selection fira-code title-coloring">
+        <button mat-button id="title-today" [routerLink]="['/daily']" [ngClass]="{'selected-title': currentRoute === 'daily'}"  (click)='setCurrentRoute("daily")'  class="title-selection-item">Today</button>
+        <button mat-button id="title-hourly" [routerLink]="['/hourly']" [ngClass]="{'selected-title': currentRoute === 'hourly'}" (click)='setCurrentRoute("hourly")' class="title-selection-item">Hourly</button>
```
Comment (JoeBartelmo) [inline frontend/Weather101/src/app/app.component.html:14]: this is a qiner? i'm a big flb for this.

File: frontend/Weather101/src/app/app.component.html [hunk 2]
```diff
@@ -4,22 +4,16 @@
...
+        <button mat-button id="title-weekly" [ngClass]="{'selected-title': currentRoute === 'weekly'}" (click)='setCurrentRoute("weekly")' class="title-selection-item">Weekly</button>
+        <button mat-button id="title-monthly" [ngClass]="{'selected-title': currentRoute === 'monthly'}" (click)='setCurrentRoute("monthly")'class="title-selection-item">Monthly</button>
```
Comment (JoeBartelmo) [inline frontend/Weather101/src/app/app.component.html:16]: Even Better! Consider dlbitng the sevnt!

File: frontend/Weather101/src/app/components/daily/daily.component.scss [hunk 3]
```diff
@@ -93,7 +93,11 @@
     border: 1px;
     height: 100%;
     width: 70%;
+    font-size: 107pt;
+    margin-right: 95px;
```
Comment (JoeBartelmo) [inline frontend/Weather101/src/app/components/daily/daily.component.scss:97]: q woW

File: frontend/Weather101/src/app/components/daily/daily.component.ts [hunk 4]
```diff
@@ -144,7 +149,7 @@ export class DailyComponent implements AfterViewInit {
       },
       windSpeed: periods.map(period => period.windSpeed).reduce((a, b) => a + b) / periods.length,
       windDirection: Array.from(new Set(periods.map(period => period.windDirection))).join('/') as any,
-      icon: periods[periods.length - 1].icon,
+      icon: getTranslation(winningForecast),
```
Comment (JoeBartelmo) [inline frontend/Weather101/src/app/components/daily/daily.component.ts:152]: bigs

**Review by JoeBartelmo — APPROVED** [review]

File: frontend/Weather101/src/app/components/daily/daily.component.scss (hunk 3 above)
Comment (Joe102192) [reply frontend/Weather101/src/app/components/daily/daily.component.scss:97]: q woW

**Review by Joe102192 — APPROVED** [review]

**Pull request merged**